cohortextractor/measure.py

```diff
@@ -1,5 +1,12 @@
+import numpy
+
+SMALL_NUMBER_THRESHOLD = 5
+
+
 class Measure:
-    def __init__(self, id, denominator, numerator, group_by=None):
+    def __init__(
+        self, id, denominator, numerator, group_by=None, small_number_suppression=False
```

Comment:
Should this default to True? I feel we should have small num supression as default behaviour?

Reply:
The feedback I got from talking to various people was that suppressing like this is just one way of getting around the problem of small numbers. Others are re-categorizing, rounding and sampling. So it's helpful to have this available as a tool, but we don't necessarily want to apply it all the time.